src/App.jsx

```diff
@@ -8,21 +8,23 @@ import ExponentSix from "./components/ExponentSix";
 
 
 function App () {
+  const [count, setCount] = useState(0);
```

Comment:
Perfecto, esto es lo que conocemos como "lifting the state up". El estado antes estaba en el elemento hijo y ahora lo tenemos en el padre accesible al resto de componentes.